Hints.py

```diff
@@ -1040,6 +1040,8 @@ def get_important_check_hint(spoiler, world, checked):
                 or location.item.name == 'Ocarina' and not world.settings.shuffle_ocarinas
                 or 'Bean' in location.item.name and not world.settings.shuffle_beans
                 or location.item.name == 'Weird Egg' and not world.settings.shuffle_child_trade == 'shuffle')
+                or location.item.name == 'Biggoron Sword'
+                or location.item.name == 'Double Defense'
```

Comment:
I don't think this does what you think it does - you inserted this into a conditional that removes items from counting as major items.

Reply:
It does, if you check the others they have nots to exclude them. This will add them.

Reply:
Looking at it again, the `not()` I thought this was a part of ends the line before your additions. This conditional is insane but addressing that is not really in the scope of this PR.